pkg/engine/lifecycletest/pulumi_test.go

```diff
@@ -2175,6 +2206,18 @@ func TestProviderPreviewUnknowns(t *testing.T) {
 			}))
 		}
 
+		outs, _, _, err := monitor.Call("pkgA:m:typA/methodA", resource.PropertyMap{
```

Comment:
Do we need to do something similar for `Invoke` as well?  Or does `monitor.Call` call that under the hood somehow?

Reply:
> Do we need to do something similar for Invoke as well

We do! Let me add that quickly.